compass/ocean/tests/spherical_harmonic_transform/__init__.py

```diff
@@ -0,0 +1,20 @@
+from compass.testgroup import TestGroup
+
+from compass.ocean.tests.spherical_harmonic_transform.qu_convergence \
+    import QUConvergence
```

Comment:
```suggestion
    import QuConvergence
```

Reply:
I knew I was going to miss one of those. Thanks!